aviary/interface/methods_for_level2.py

```diff
@@ -1702,7 +1702,7 @@ def add_design_variables(self):
             optimize_mass = self.pre_mission_info.get('optimize_mass')
             if optimize_mass:
                 self.model.add_design_var(Mission.Design.GROSS_MASS, units='lbm',
-                                          lower=100.e2, upper=200.e3, ref=135.e3)
+                                          lower=100.e2, upper=900.e3, ref=135.e3)
```

Comment:
in the past we had the upper limit set to `None` to prevent this issue.
@johnjasa should we go back to None?

Reply:
I don't think so as IPOPT benefits from having a reasonable upper limit for its auto-scaling